### PR TITLE
Add CI/CD pipeline

### PR DESCRIPTION
**Pull request by kkinch**
Publish swagger API documentation to CloudDocs.

   https://clouddocs.f5networks.net/cloud/openstack/v1/wafaas/api/

- Put gh-pages files in docs/_static folder.
- Change travis.yml language from node_js to Python and add publish to clouddocs info.
- Clouddocs css overrode swagger css. Edit .scheme-container to remove padding element.
- Remove <html> tags from index.html.
- Send swagger to staging on clouddocs until ready for world.

### CODE REVIEW

File: docs/_static/swagger-ui.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.swagger-ui{
```
Comment (zongzw) [inline docs/_static/swagger-ui.css]: These files(favicon-*.png index.html openapi.json swagger-*) can be generated by script dynamically. 
@zhaoqin-github could you please extract the _static file generation functions out of publish_api_ref.sh.

**Comment by zongzw** [comment]
Thanks, @kkinch to help set up the docs pipeline. Great efforts.

**Comment by zhaoqin-github** [comment]
@kkinch If we plan to publish api document to F5 cloudocs, I think that can be a CI task for tagged branch. For example, when we release a new version, we need to create a new tag, and Travis build need to generate API document from source code on that tag branch, then upload the doc to F5 clouddocs.

We expect to always generate docs from source code, instead of saving generated docs in github repo, in order to ensure the API docs are always consistent with the code. We may have two places to save API docs. One is gh-pages. The docs in gh-pages are generated from latest source code. Another can be F5 clouddocs, which is the place to save docs generated from released version of source code.

**Comment by kkinch** [comment]
@zhaoqin-github Would it be possible for you to adapt your current .travis-ci file based on my PR to meet your tagging requirements? Once completed, I could just remove my PR. All of the AWS keys are in place for you to publish. No problems leaving your gh-pages branch too!

Thank you!

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+:product: F5 ADCaaS for OpenStack
```
Comment (zhaoqin-github) [inline docs/index.rst]: :product: F5 OpenStack Services

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+:type: concept
+
+.. _f5-adcaas-home:
+
+F5 ADCaaS for OpenStack
```
Comment (zhaoqin-github) [inline docs/index.rst]: F5 OpenStack Services

**Comment by zhaoqin-github** [comment]
@kkinch  I create #294 to publish the doc. Close this pr.

**Pull request closed without merging**